doc/languages-frameworks/go.section.md

```diff
@@ -11,7 +11,8 @@ The function `buildGoModule` builds Go programs managed with Go modules. It buil
 
 In the following is an example expression using `buildGoModule`, the following arguments are of special significance to the function:
 
-- `vendorHash`: is the hash of the output of the intermediate fetcher derivation. `vendorHash` can also take `null` as an input. When `null` is used as a value, rather than fetching the dependencies and vendoring them, we use the vendoring included within the source repo. If you'd like to not have to update this field on dependency changes, run `go mod vendor` in your source repo and set `vendorHash = null;`
+- `vendorHash`: is the hash of the output of the intermediate fetcher derivation. `vendorHash` can also take `null` as an input. When `null` is used as a value, rather than fetching the dependencies and vendoring them, we use the vendoring included within the source repo. If you'd like to not have to update this field on dependency changes, run `go mod vendor` in your source repo and set `vendorHash = null;`  
+  (On first run, you can set it to `lib.fakeSha256` and run the build to see the actual hash)
```

Comment:
There is a section in the manual where it is explained why and when to use which hash placeholder. It would be great if you link to that here, if you find time.

Otherwise, here's a suggestion for more brevity and accessibility (not everyone can see):

```suggestion
  To obtain the actual hash, set `vendorHash = lib.fakeSha256;` and run the build.
```

Reply:
Added a link to https://nixos.org/manual/nixpkgs/stable/#sec-source-hashes :)